### PR TITLE
Fixes editor device linking tool

### PR DESCRIPTION
**Pull request by Bod9001**
it was forgetting to set the interface master dirty

### CODE REVIEW

File: UnityProject/Assets/Scripts/EditorButInAssetsAssembly/DeviceLinker.cs [hunk 1]
```diff
@@ -179,7 +179,7 @@ private int LinkSlaves(bool relinkConnected)
 				}
 
 
-				if (slave.Master != null && relinkConnected == false) return false;
+				//if (slave.Master != null && relinkConnected == false) return false;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/EditorButInAssetsAssembly/DeviceLinker.cs]: why is this check not needed anymore?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/EditorButInAssetsAssembly/DeviceLinker.cs]: it seems to cause issues if stuff is partially linked, so might as well do re-complete link every time

Comment (corp-0) [reply UnityProject/Assets/Scripts/EditorButInAssetsAssembly/DeviceLinker.cs]: I see, can you remove it? Don't comment stuff out, someone in the future might be wondering why is it commented out and think uncommenting might solve something

Comment (Bod9001) [reply UnityProject/Assets/Scripts/EditorButInAssetsAssembly/DeviceLinker.cs]: ok

**Review by corp-0 — APPROVED** [review]

**Pull request merged**